packages/@aws-cdk/aws-ecs/test/ec2/test.ec2-task-definition.ts

```diff
@@ -472,6 +472,24 @@ export = {
       test.done();
     },
 
+    "warns when setting containers from ECR repository using fromRegistry method"(test: Test) {
+      // GIVEN
+      const stack = new cdk.Stack();
+
+      const taskDefinition = new ecs.Ec2TaskDefinition(stack, 'Ec2TaskDef');
+
+      // WHEN
+      const container = taskDefinition.addContainer("web", {
+        image: ecs.ContainerImage.fromRegistry("ACCOUNT.dkr.ecr.REGION.amazonaws.com/REPOSITORY"),
```

Comment:
I would like to see a test that an ECR registry that is created in the same stack is properly recognized in this way as well.

Reply:
Added but not sure if I correctly understand what you meant.